src/Discord.Net.Rest/API/Rest/EditWebhookMessageParams.cs

```diff
@@ -0,0 +1,22 @@
+#pragma warning disable CS1591
+using Newtonsoft.Json;
+
+namespace Discord.API.Rest
+{
+    [JsonObject(MemberSerialization = MemberSerialization.OptIn)]
+    internal class EditWebhookMessageParams
```

Comment:
Rename it to `ModifyWebhookMessageParams` and follow the same format as [ModifyMessageParams](https://github.com/discord-net/Discord.Net/blob/dev/src/Discord.Net.Rest/API/Rest/ModifyMessageParams.cs) (optional content, missing `get; set;`, no constructor).